### DISCUSSION TITLE
Conditional render of properties in backoffice

### DISCUSSION THREAD

**Discussion by albinbrengesjo** [umbraco/Umbraco-CMS#13562] [Features and ideas]
Hi,

Would be benefitial to render properties conditionally in backoffice. The feature exists in Umbraco forms when arranging inputs, would be cool with the same feature for backoffice users. See exemple below.

https://user-images.githubusercontent.com/101869333/207022636-09ff3c0d-5458-4580-8721-0d45f64d3ea3.mov

**Comment by skttl** [comment]
[Theres a package for that ;)](https://marketplace.umbraco.com/package/our.umbraco.conditionaldisplayers)

**Comment by bergmania** [comment]
It's a great idea, but it will not be something we have time to look into before the new backoffice.  Until then, use a package.

**Reply by jdpnielsen** [reply]
Is there any movement on this front with Bellissima? A package might be fine for conditional display, but it does not solve requirements around conditional validation rules.

My usecase is specifically a 'media block' where an editor can either select an item from the media library or a video using the [TwentyThree](https://marketplace.umbraco.com/package/limbo.umbraco.twentythree) picker. In this case i want to make sure that the user has selected at least one item in either picker - this could be modelled as a dropdown controlling which picker is required.

**Reply by bergmania** [reply]
No movement. We try not to scope creep as the backoffice is already a moving target. So this will not be part of a first version.

**Comment by gerrit-de-heij** [comment]
Is there an update on this? The new backoffice is out for some time now and I think that this should be a native feature of Umbraco in my opinion.

**Reply by Rockerby** [reply]
The package @skttl linked to above is available for v17 -- https://marketplace.umbraco.com/package/our.umbraco.conditionaldisplayers

I've been working on a new version of this to make it more of a 1st class citizen within Umbraco; always happy to receive some testing / support! https://github.com/Rockerby/Umbraco.Community.ConditionalProperties

**Comment by nielslyngsoe** [comment]
Hi @albinbrengesjo and @gerrit-de-heij 

This is a native feature of the CMS, from a UI perspective.

It can be utilized via user Group permissions, so some users can see/edit more fields and others.

There is no admin interface for configuring conditions based on Property Values, but this can surely be done. It is a runtime api, so it can change at any time, and that change will take effect.

You can read about how you can write your own logic as part of this article:
https://docs.umbraco.com/umbraco-cms/extend-your-project/backoffice-extensions/property-level-ui-permissions

I dont think the referenced package above uses this API, so there is an opportunity to build a very nice pacage that provides UI for admins to configure this via UI, instead of code. But for now we are not aiming to build that; instead, we have the API, and you can make it react as you need to.
